extensions/datastores/foundationdb/src/main/java/org/locationtech/geowave/datastore/foundationdb/FoundationDBDataStore.java

```diff
@@ -9,20 +9,22 @@
 import java.io.IOException;
 
 public class FoundationDBDataStore extends BaseMapReduceDataStore implements Closeable {
```

Comment:
This in all likelihood does not need to implement Closeable ... RocksDB is the only closeable datastore and it is only that way because there are reasons to leave native resources open outside of the scope of the various methods such as the 6 create (metadata) reader/writer/deleter methods.  For other datastores the closeable resources are all allocated and self-contained within the scope of the methods so there is no global need to close resources on the datastore as a whole (and this isolation is ideal as long as it wouldn't be a performance concern in FDB).

Reply:
Fair enough. I've removed this for now.